jumpscale/sals/nginx/templates/website.conf

```diff
@@ -14,6 +14,10 @@
     {% if website.domain %}
         server_name                 ~^(www\.)?{{ website.domain }}$;
     {% endif %}
+        error_page 401 = @error401;
```

Comment:
I think we need to deal with 403 too, see admin_only, it can return 403:

https://github.com/threefoldtech/js-sdk/blob/development_locations_auth/jumpscale/packages/auth/bottle/auth.py#L222